apis/vshn/v1/common_types.go

```diff
@@ -178,6 +178,8 @@ type VSHNScheduleStatus struct {
 
 // Security defines the security of a service
 type Security struct {
+	// AllowAll allows the service to be accessible from all namespaces, this supersedes the AllowedNamespaces field
```

Comment:
Should we maybe set the default to `false` explicitly via a controller-gen comment?

Reply:
no problem from my side